libs/common/src/enums/feature-flag.enum.ts

```diff
@@ -35,6 +35,7 @@ export enum FeatureFlag {
   AccessIntelligence = "pm-13227-access-intelligence",
   Pm13322AddPolicyDefinitions = "pm-13322-add-policy-definitions",
   LimitCollectionCreationDeletionSplit = "pm-10863-limit-collection-creation-deletion-split",
+  CriticalApps = "critical-apps",
```

Comment:
According to the [server PR](https://github.com/bitwarden/server/pull/4975/files#diff-94998af117f486ce85a16234b1bf36dd8c918651ff4d5c851927d203f5120e4eR151) this should be
```suggestion
  CriticalApps = "pm-14466-risk-insights-critical-application",
```